src/main/java/org/spongepowered/common/item/inventory/util/ContainerUtil.java

```diff
@@ -169,7 +169,7 @@ public static void performBlockInventoryDrops(WorldServer worldServer, double x,
 
                     while (!itemStack.isEmpty())
                     {
-                        int i = RANDOM.nextInt(21) + 10;
```

Comment:
I think this is supposed to stay as-is. It was copied from `InventoryHelper.spawnItemStack()`.

Reply:
This shouldn't be breaking your code because it loops until the original item stack is empty.